web/themes/custom/votegov/src/sass/_uswds-settings.scss

```diff
@@ -165,6 +165,8 @@ https://designsystem.digital.gov/documentation/settings/
   $theme-column-gap-lg: 6,
   $theme-column-gap-mobile: 5,
   //$theme-column-gap-desktop: 7,
+  $theme-site-margins-width:          3,
```

Comment:
Update `$theme-site-margins-width` to 10 which is 5rem.
Add `$theme-site-margins-breakpoint` to `'tablet'`